src/main/resources/requests/accountManagementRequests.ts

```diff
@@ -0,0 +1,23 @@
+import {accountManagementApi} from './utils/axiosConfig';
+
+//const superagent = require('superagent');
+
+export class AccountManagementRequests {
+  public accountManagementApi = process.env.ACCOUNT_MANAGEMENT_URL || 'https://pip-account-management.staging.platform.hmcts.net';
```

Comment:
can you not use axios one? this had to be added as axios doesn't work with superagent

Reply:
used axios